docs/Installing.asciidoc

```diff
@@ -141,6 +141,24 @@ httpsonly = 0
 Since version _4.5.1512500474.437cc1c7_ of openQA, PostgreSQL is used as the
 database.
 
+Install Postgres if not yet installed:
+[source,sh]
+--------------------------------------------------------------------------------
+# openSUSE
+zypper in postgresql postgresql-server
+# Fedora
```

Comment:
As this installing document is about packages - this is wrong. You should install openQA-local-db and that does pull the requires and sets up the database.